remo/reports/static/reports/js/edit_ng_report.js

```diff
@@ -51,4 +51,6 @@
     // set date picker defaults
     initDatePicker();
 
+    // set the timezone here
```

Comment:
I personally think  this is an unnecessary comment as the code already says that. I think it would be better to say *why* we need to set this in a hidden field and maybe even reference the bug.